### PR TITLE
chore: fix apk path in script

### LINKED ISSUE #661: fix apk path in script

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #661

### CODE REVIEW

File: scripts/upload-apk.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ for file in android*; do
     if [[ $file =~ ".aab" ]]; then
         mv $file badge-magic-$TRAVIS_BRANCH-$file
     else
-        mv $file badge-magic-$TRAVIS_BRANCH-${file:4}
+        mv $file badge-magic-$TRAVIS_BRANCH-${file:8}
```
Comment (iamareebjamal) [inline scripts/upload-apk.sh]: Hmm I thought you would have realized how brittle this is

Comment (adityastic) [reply scripts/upload-apk.sh]: I just noticed that the names are not correct in the apk branch when you posted a link in #660 xD

Comment (iamareebjamal) [reply scripts/upload-apk.sh]: Yes, it is because this method of renaming is error prone

Comment (adityastic) [reply scripts/upload-apk.sh]: Let me find the first index of - and substring. I'll update it in sometime

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**